includes/results.inc

```diff
@@ -847,8 +847,8 @@ class IslandoraSolrFacets {
     foreach ($results as $bucket => $count) {
       $facet_results[] = array(
         'count' => $count,
-        'filter' => islandora_solr_lesser_escape($facet_field) . ':"' .
-        islandora_solr_lesser_escape($bucket) . '"',
+        'filter' => islandora_solr_facet_escape($facet_field) . ':"' .
```

Comment:
Doesn't really hurt I guess... Just seems like encountering double-quotes in field names shouldn't happen. *mumbles incoherently*

... Eh... guess I'm fine with it like this.